src/components/purchase-complete/index.js

```diff
@@ -81,19 +115,21 @@ const PurchaseComplete = ({
                 isActive ?
                     (currentUserTicket && requireExtraQuestions) ?
                         <>
-                            <span>{T.translate('purchase_complete_step.finish_now_label')}</span>
-                            <CTAButton cta={goToExtraQuestions} clear={clearWidgetState}
-                                       close={closeWidget}>{T.translate('purchase_complete_step.finish_now_button')}</CTAButton>
+                            <span>{orderComplete1stParagraph ? orderComplete1stParagraph : T.translate('purchase_complete_step.finish_now_label')}</span>
+                            <CTAButton cta={goToExtraQuestions} clear={clearWidgetState} close={closeWidget}>
+                                {orderCompleteButtonText ? orderCompleteButtonText : T.translate('purchase_complete_step.my_orders_button')}
```

Comment:
this should be https://github.com/fntechgit/summit-registration-lite/blob/2d8a39f6f09729c69131c526732ccc6bff598fcc/src/components/purchase-complete/index.js#L86